setup

```diff
@@ -6,11 +6,18 @@ set -e
 #
 # --- Based Off of/Ripped from https://github.com/FasterMelee/FasterMelee-installer/ Thank you Faster Melee Team!
 # ---
+ONLINEMASTER=$(curl -Ls https://github.com/Birdthulu/FPM-Installer/raw/master/setup | md5sum)
+CURRENTMASTER=$(md5sum setup)
+##Cut these down to just 32 hex chars
 
-if [ -f "./setup" ] && [ -z "${IN_NIX_SHELL++}" ]; then
-	echo "You are running a hosted copy of the script. Please *delete the local file 'setup'* and run"
+ONLINEMASTER=$(echo $ONLINEMASTER | cut -c -32)
+CURRENTMASTER=$(echo $CURRENTMASTER | cut -c -32)
+echo $ONLINEMASTER
```

Comment:
Change to:
```
echo "Your Version:" $ONLINEMASTER
echo "Online Version:" $CURRENTMASTER
```

Reply:
I flipped those, since $ONLINEMASTER refers to the one on github.